File: 00_IntroProgramacionYMoviles/Hello.kt
package com.example.myapplication
fun CompararNumeros (A : Int, B: Int) {
    if (A==B) {
        print ("Son Iguales")
    } else {
        if (A > B) {
            print ("A es mayor que B")
        } else {
            print ("B es mayor que A")
        }
    }
}
fun main() {
    print("Escribe el primer numero (A):")
    val As = readLine()!!
    val A= Integer.parseInt(As)
    print("Escribe el segundo numero (B):")
    val Bs = readLine()!!
    val B= Integer.parseInt(Bs)
    CompararNumeros(A,B)
}

File: 00_IntroProgramacionMovil/Hello.py
def CompararNumeros(A,B):
	if (A==B):
		print ("Son Iguales ")
	else:
		if (A>B):
			print ("A es mayor que B ")
		else:
			print ("B es mayor que A ")

As = input ("Escribe el primer numero (A): ")
A = int(As)
Bs = input ("Escribe el segundo numero (B): ")
B = int(Bs)
CompararNumeros(A,B)
